app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,12 @@
+#welcome {
+  position: relative;
+}
+
+.welcome_text {
+  width: 50%;
```

Comment:
Properties should be ordered bottom, height, left, margin, overflow, position, right, top, width